src/views/Home.jsx

```diff
@@ -6,6 +6,19 @@ export function Home(props) {
 			<button type="button" onClick={props.handleClick}>
 				Create list
 			</button>
+			<p>- or -</p>
+			<p>Join an existing shopping list by entering a three word token</p>
+			<form>
+				<label htmlFor="token">
+					<input
+						type="text"
+						id="token"
+						name="token"
+						placeholder="Three word token"
```

Comment:
I'd recommend giving this input [a `required` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#required) to prevent the user from submitting an empty form. Without it, the user can now click on "Join an existing list" without having input a three word token, resulting in a Firebase error:
<img width="448" alt="image" src="https://user-images.githubusercontent.com/58532369/196314346-09b238a5-cb81-42d2-a0f9-755fd06f8bad.png">

Reply:
Done